src/lib/seam/SeamProvider.tsx

```diff
@@ -70,42 +70,42 @@ export function SeamProvider({
   useSeamStyles({ disabled: disableCssInjection, unminified: unminifiyCss })
   useSeamFont({ disabled: disableFontInjection })
 
-  const value = useMemo(() => {
+  const seamProviderContextValue = useMemo(() => {
     const context = createSeamContextValue(props)
     if (
       context.client == null &&
       context.publishableKey == null &&
       context.clientSessionToken == null
     ) {
-      return defaultSeamContextValue
+      return getDefaultSeamContextValue()
     }
     return context
   }, [props])
 
   if (
-    value.client == null &&
-    value.publishableKey == null &&
-    value.clientSessionToken == null
+    seamProviderContextValue.client == null &&
+    seamProviderContextValue.publishableKey == null &&
+    seamProviderContextValue.clientSessionToken == null
   ) {
     throw new Error(
       `Must provide either a Seam client, clientSessionToken, or a publishableKey.`
     )
   }
 
-  const { Provider } = seamContext
-
   return (
     <div className='seam-components'>
       <QueryClientProvider
         client={queryClient ?? globalThis.seamQueryClient ?? defaultQueryClient}
       >
-        <Provider value={value}>{children}</Provider>
+        <seamContext.Provider value={seamProviderContextValue}>
+          {children}
+        </seamContext.Provider>
```

Comment:
This matches the existing naming convention in this project when working with providers.

```suggestion
        <Provider value={seamProviderContextValue}>
          {children}
        </Provider>
```

Reply:
I'm probably going to close this PR and go upstream to seamapi (fair), but this convention of having ambiguous "value" and "Provider" thrown around isn't a good naming pattern I think- `value`, like `data` shouldn't be a var. If the file was less complex it might be passable but this file is really complex